### ISSUE TITLE
[5.0] ansible-lint trying ansible syntax-check on requirements.yaml - 'src' is not a valid attribute for a Play

### ISSUE THREAD

**Issue by juju4** [ansible-community/ansible-lint#1388]
##### Summary
ansible-lint in molecule lint step is reporting syntax error associated to ansible playbooks but on the requirements.yaml which is invalid

https://github.com/juju4/ansible-falco/runs/1942564726?check_suite_focus=true#step:5:317
```

Molecule default > lint
  COMMAND: set -e
  yamllint .
  ansible-lint -x 204
  flake8
  
  Loading custom .yamllint config file, this extends our internal yamllint config.
  WARNING  Listing 1 violation(s) that are fatal
  syntax-check: 'src' is not a valid attribute for a Play
  Warning: cule/default/requirements.yml:3:3 [WARNING]: provided hosts list is empty, only localhost is available. Note that
  the implicit localhost does not match 'all'
  ERROR! 'src' is not a valid attribute for a Play
  
  The error appears to be in '/home/runner/work/ansible-falco/ansible-falco/juju4.falco/molecule/default/requirements.yml': line 3, column 3, but may
  be elsewhere in the file depending on the exact syntax problem.
  
  The offending line appears to be:
  
  
  - src: https://github.com/juju4/ansible-redhat-epel/
    ^ here
  
  
  Error: syntax-check 'src' is not a valid attribute for a Play
  WARNING  Replaced deprecated tag '106' with 'role-name' but it will become an error in the future.
  You can skip specific rules or tags by adding them to your configuration file:
  # .ansible-lint
  warn_list:  # or 'skip_list' to silence them completely
    - syntax-check  # Ansible syntax check failed
  Finished with 1 failure(s), 0 warning(s) on 35 files.
  CRITICAL Lint failed with error code 2
```

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible 2.10.6
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/runner/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /opt/hostedtoolcache/Python/3.9.1/x64/lib/python3.9/site-packages/ansible
  executable location = /opt/hostedtoolcache/Python/3.9.1/x64/bin/ansible
  python version = 3.9.1 (default, Jan 25 2021, 10:38:32) [GCC 9.3.0]
ansible-lint 5.0.1
```

- ansible installation method: one of source, pip, OS package
- ansible-lint installation method: one of source, pip, OS package

##### OS / ENVIRONMENT
Ubuntu 20.04 from Github

**Comment by ssbarnea** [comment]
This no longer happens with code from master branch.
